group_vars/rabbitmq/public.yml

```diff
@@ -6,4 +6,4 @@ SANITY_CHECK_LIVE_PORTS:
     port: 15671
     message: "RabbitMQ management does not respond on port 15671."
 
-TARSNAP_BACKUP_FOLDERS: /etc/rabbitmq /var/lib/rabbitmq /etc/letsencrypt
+TARSNAP_BACKUP_FOLDERS: /usr/local/lib/rabbitmq/backup /etc/rabbitmq /etc/letsencrypt
```

Comment:
I think "scoping" is the wrong word to describe how Ansible resolves variables.  There is only one scope, and everything is dumped into that single scope (in a [defined order](http://docs.ansible.com/ansible/playbooks_variables.html#variable-precedence-where-should-i-put-a-variable)) before any inline Jinja2 templates are rendered.  This process results in a whole lot of, well, unexpected behaviour, to put it friendly, e.g. if you conditionally include a role and the condition is false, you will see its variables anyway.  But at least it has the convenient effect that you can see all variables everywhere, and they have the same value everywhere (until you change them with `set_fact`, that is).